module-cheatsheets/scRNA-seq-cheatsheet.md

```diff
@@ -39,19 +39,29 @@ Read the `SingleCellExperiment` package documentation (and e-book) [**here**](ht
 | `SingleCellExperiment` | [`SingleCellExperiment()`](https://bioconductor.org/packages/release/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html) | Single Cell Experiment| Creates a `SingleCellExperiment` object  |
 | `SingleCellExperiment`| [`colData()`](https://bioconductor.org/packages/release/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#2_Creating_SingleCellExperiment_instances) | Column Data | Extracts and stores cell-level metadata that describes features of the `SingleCellExperiment` object|
 | `SingleCellExperiment`| [`rowData()`](https://bioconductor.org/packages/release/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#2_Creating_SingleCellExperiment_instances)  | Row Data   | Extracts and stores gene-level metadata that describes features of the `SingleCellExperiment` object|
+| `SingleCellExperiment`| [`assays()` and `assay()`](https://bioconductor.org/packages/release/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#2_Creating_SingleCellExperiment_instances)  | Assay(s) | Extracts or assigns a given assay(s) from a `SingleCellExperiment` object|
+| `SingleCellExperiment`| [`assayNames()`](https://bioconductor.org/packages/release/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html#3_Adding_low-dimensional_representations)  | Assay names  | Returns a vector of the names of all assays in a `SingleCellExperiment` object|
```

Comment:
While `#3_Adding_low-dimensional_representations` is the first place in that doc that `assayNames()`  appears, I'm not sure it is a good place to link for the docs...

Reply:
Agreed...Done in https://github.com/AlexsLemonade/training-modules/pull/663/commits/b585b5fb43da26ecc0cbfdd64b24672b91151aba